ui/game/src/view/status.ts

```diff
@@ -85,8 +85,6 @@ export default function status(ctrl: Ctrl): string {
           return noarg('threeChecks') + winnerSuffix;
       }
       return noarg('variantEnding') + winnerSuffix;
-    case 'unknownFinish':
```

Comment:
`unknownFinish` still exists on the server side, and will probably reach this `switch`. I think it's mostly for imported games.